markets/perps-market/test/integration/Market/MarketConfigration.test.ts

```diff
@@ -61,34 +62,110 @@ describe('MarketConfiguration', async () => {
     );
   });
 
-  it('owner can set all market configurations properties', async () => {
-    await systems()
-      .PerpsMarket.connect(marketOwner)
-      .addSettlementStrategy(marketId, fixture.settlementStrategy);
-    await systems()
-      .PerpsMarket.connect(marketOwner)
-      .setSettlementStrategyEnabled(marketId, 0, fixture.settlementStrategy.disabled);
-    await systems()
-      .PerpsMarket.connect(marketOwner)
-      .setOrderFees(marketId, fixture.orderFees.makerFee, fixture.orderFees.takerFee);
-    await systems()
-      .PerpsMarket.connect(marketOwner)
-      .setMaxMarketValue(marketId, fixture.maxMarketValue);
-    await systems()
-      .PerpsMarket.connect(marketOwner)
-      .setFundingParameters(marketId, fixture.skewScale, fixture.maxFundingVelocity);
-    await systems()
-      .PerpsMarket.connect(marketOwner)
-      .setLiquidationParameters(
-        marketId,
-        fixture.initialMarginFraction,
-        fixture.maintenanceMarginFraction,
-        fixture.liquidationRewardRatioD18,
-        fixture.maxLiquidationLimitAccumulationMultiplier
-      );
-    await systems()
-      .PerpsMarket.connect(marketOwner)
-      .setLockedOiPercent(marketId, fixture.lockedOiPercent);
+  it('owner can set all market configurations properties and events are emitted', async () => {
```

Comment:
in general, better to split these into separate `it` blocks but since these are events,  if even one of these events fails, it'll bubble it up so not a huge deal here.